silkworm/infra/common/environment.cpp

```diff
@@ -84,4 +84,10 @@ void Environment::set_pre_verified_hashes_disabled() {
     environment["DISABLE_PRE_VERIFIED_HASHES"] = "1";
 }
 
+std::string Environment::get(const std::string& var_name) {
+    auto environment = boost::this_process::environment();
```

Comment:
what does boost give us here compared to getenv/setenv?

Reply:
> what does boost give us here compared to getenv/setenv? (just curious, no need to change the code)

At least no warning about `std::getenv` being thread-unsafe 😃